src/compiler/scala/tools/nsc/classpath/ZipArchiveFileLookup.scala

```diff
@@ -81,6 +81,6 @@ trait ZipArchiveFileLookup[FileEntryType <: ClassRepresentation] extends Efficie
 
 
   protected def createFileEntry(file: FileZipArchive#Entry): FileEntryType
-  protected def isRequiredFileType(file: AbstractFile): Boolean
+  protected def isRequiredFileType(file: AbstractFile, fileExists: String => Boolean): Boolean
```

Comment:
```suggestion
  protected def isRequiredFileType(file: AbstractFile, siblingExists: String => Boolean): Boolean
```

If you're going to push once more anyway

Reply:
good spot thanks!